File: Chronos.App/ClientApp/src/app/models/rules/task-rules.ts
import { Variable } from "./variable";

export class TaskRules {
    title = '';
    description = '';
    titleVariables: Variable[] = [];
    tags: string[] = [];
    defaultTags: string[] = [];

    constructor(obj?: Partial<TaskRules>) {
        if (obj)
            Object.assign(this, obj);
    }
}

File: Chronos.Domain/Entities/FeatureRules/TaskRules.cs
﻿using System.Runtime.Serialization;

namespace Chronos.Domain.Entities.FeatureRules
{
    [DataContract]
    public class TaskRules
    {
        [DataMember]
        public string Title { get; set; }

        public TitleVariable[] TitleVariables { get; set; }

        [DataMember]
        public string Description { get; set; }

        [DataMember]
        public string[] Tags { get; set; }

        public string[] DefaultTags { get; set; }
    }
}
